tests/lib/Migrations/Version20210913225128.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GroupLife\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210913225128 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Initialize DB';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('CREATE TABLE `schedules` (id INT AUTO_INCREMENT NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('
+            CREATE TABLE `rules` (
+                id INT AUTO_INCREMENT NOT NULL, 
+                rules VARCHAR(255), 
```

Comment:
What do you plan to save in this string?

Reply:
I planned to keep the rules for the formation of the schedule

Reply:
The most important is how to store the business rules: weekdays, dates, times etc. Do you have any suggestions?

Reply:
I think we need one more teble 'type' to define the type of rules and store the rules in a string